monix-eval/shared/src/main/scala/monix/eval/Task.scala

```diff
@@ -3517,36 +3517,46 @@ object Task extends TaskInstancesLevel1 {
     implicit bf: BuildFrom[M[A], B, M[B]]): Task[M[B]] =
     TaskSequence.traverse(in, f)(bf)
 
+  /** Alias for [[parSequence]] */
+  @deprecated("Use parSequence", "3.2.0")
+  def gather[A, M[X] <: Iterable[X]](in: M[Task[A]])(implicit bf: BuildFrom[M[Task[A]], A, M[A]]): Task[M[A]] =
```

Comment:
We should put deprecated methods to `TaskDeprecated.Companion` to clean normal API

There is also slightly different convention for description with
```
/**
   * DEPRECATED — please use XXX
   */
```